app/views/layouts/_footer.html.erb

```diff
@@ -12,5 +12,9 @@
 	<a href="https://prochoice.org/">National Abortion Federation</a> -- <a href="tel:800-772-9100"><span class="glyphicon glyphicon-earphone" aria-hidden="true"></span> 800-772-9100</a><br>
 	<a href="http://onlinefaxes.com">onlinefaxes.com</a>
 </div><!-- end container -->
+<div class="container text-center margin-bottom-sm">
+	<img id="logo" alt="DCAF" src="/assets/dcaf-letter-blocks.svg">
```

Comment:
When we push this into production, these'll break because of the asset pipeline, the cause of and solution to most of rails' problems. Can you switch these to the rails helper `image_tag`? Docs are here: http://api.rubyonrails.org/classes/ActionView/Helpers/AssetTagHelper.html#method-i-image_tag